### PR TITLE
Change finals from docblock to prefix

### PR DESCRIPTION
**Pull request by wbloszyk**

### CODE REVIEW

**Review by core23 — DISMISSED** [review]

**Review by core23 — APPROVED** [review]

File: tests/Block/Service/AbstractCategoriesBlockServiceTest.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ protected function setUp(): void
         $this->twig = $this->createMock(Environment::class);
         $this->contextManager = $this->createMock(ContextManagerInterface::class);
         $this->categoryManager = $this->createMock(CategoryManagerInterface::class);
-        $this->categoryAdmin = $this->createMock(CategoryAdmin::class);
+        $this->categoryAdmin = new CategoryAdmin('code', 'class', 'controller', $this->contextManager);
```
Comment (franmomu) [inline tests/Block/Service/AbstractCategoriesBlockServiceTest.php:53]:
```suggestion
        $this->categoryAdmin = new CategoryAdmin('code', \stdClass::class, 'controller', $this->contextManager);
```

phpstan will probably complain on a higher level as `class` is not a `class-string`

File: tests/Block/Service/AbstractCollectionsBlockServiceTest.php [hunk 2]
```diff
@@ -50,7 +50,7 @@ protected function setUp(): void
         $this->twig = $this->createMock(Environment::class);
         $this->contextManager = $this->createMock(ContextManagerInterface::class);
         $this->collectionManager = $this->createMock(CollectionManagerInterface::class);
-        $this->collectionAdmin = $this->createMock(CollectionAdmin::class);
+        $this->collectionAdmin = new CollectionAdmin('code', 'class', 'controller', $this->contextManager);
```
Comment (franmomu) [inline tests/Block/Service/AbstractCollectionsBlockServiceTest.php:53]:
```suggestion
        $this->collectionAdmin = new CollectionAdmin('code', \stdClass::class, 'controller', $this->contextManager);
```

File: tests/Block/Service/AbstractTagsBlockServiceTest.php [hunk 3]
```diff
@@ -50,7 +50,7 @@ protected function setUp(): void
         $this->twig = $this->createMock(Environment::class);
         $this->contextManager = $this->createMock(ContextManagerInterface::class);
         $this->tagManager = $this->createMock(TagManagerInterface::class);
-        $this->tagAdmin = $this->createMock(TagAdmin::class);
+        $this->tagAdmin = new TagAdmin('code', 'class', 'controller', $this->contextManager);
```
Comment (franmomu) [inline tests/Block/Service/AbstractTagsBlockServiceTest.php:53]:
```suggestion
        $this->tagAdmin = new TagAdmin('code', \stdClass::class, 'controller', $this->contextManager);
```

**Review by franmomu — APPROVED** [review]

File: tests/Block/Service/AbstractCategoriesBlockServiceTest.php (hunk 1 above)
Comment (wbloszyk) [reply tests/Block/Service/AbstractCategoriesBlockServiceTest.php:53]: I will and it too phpstan upgrade PR.

**Pull request merged**